tests/test_input_output.py

```diff
@@ -111,6 +111,94 @@ def test_write_desc_input_Nones(tmpdir_factory):
     assert no_gtol  # fail test if gtol line is written
 
 
+@pytest.mark.unit
+def test_descout_to_input(tmpdir_factory):
```

Comment:
does this also test that the profiles are correct? Would be good to test both even/nonsym profiles (and profiles that have several nonzero values) to make sure that parts works correctly too

Reply:
Not right now. Ok, will do.

Reply:
I have added a simple check for pressure but no test files.